executor/builder.go

```diff
@@ -1300,6 +1300,46 @@ func (b *executorBuilder) buildMaxOneRow(v *plannercore.PhysicalMaxOneRow) Execu
 }
 
 func (b *executorBuilder) buildUnionAll(v *plannercore.PhysicalUnionAll) Executor {
+	if v.IsPointGetUnion {
```

Comment:
It would be better if we just let one PointGet hold all values instead of using a union all operator?

Reply:
I think we should keep them independent to reduce the complexity of PointGet executor.

Reply:
Just use the same plan operator. You can decide to use batched executor or not when building executor.

Reply:
If we share the same `PointGetPlan` between them, we must add some extra fields to `PointGetPlan` which will make it performance regression. The `PointGetPlan` is used so frequently so we should keep it small.